Update Docs API Digital Certificates & Domains (#64)
aziontech/docs_en: api/v3/digital-certificates/index.md

Through the Digital Certificates API you can consult, create, remove or update your SSL certificates.

> > 1. [Check the list of Digital Certificates](#consultar-lista-de-digital-certificates)

Reviewer: ```suggestion
> 1. [Check the list of Digital Certificates](#list-digital-certficates)
```

> > 2. [Check the data of a Digital Certificate id](#consultar-dados-de-um-digital-certificate-id)

Reviewer: ```suggestion
> 2. [Check the data of a Digital Certificate by id](#check-digital-certificate-id)
```

> > 3. [Check the data of a Digital Certificate name](#consultar-dados-de-um-digital-certificate-name)

Reviewer: ```suggestion
> 3. [Check the data of a Digital Certificate by name](#check-digital-certificate-name)
```

> > 4. [Deleting a Digital Certificate](#deletar-um-digital-certificate)

Reviewer: ```suggestion
> 4. [Deleting a Digital Certificate](#delete-digital-certificate)
```

> > 5. [Create a new Digital Certificate](#criar-um-novo-digital-certificate)

Reviewer: ```suggestion
> 5. [Create a new Digital Certificate](#create-digital-certificate)
```

> > 6. [Overwrite a Digital Certificate](#sobrescrever-um-digital-certificate)

Reviewer: ```suggestion
> 6. [Overwrite a Digital Certificate](#overwrite-digital-certificate)
```

> > 7. [Update fields of a Digital Certificate](#atualizar-campos-de-um-digital-certificate)

Reviewer: ```suggestion
> 7. [Update fields of a Digital Certificate](#update-fields-digital-certificate)
```
